web/src/lib/components/layouts/user-page-layout.svelte

```diff
@@ -23,7 +23,7 @@
 </header>
 
 <main
-	class="grid grid-cols-[250px_auto] relative pt-[4.25rem] h-screen bg-immich-bg dark:bg-immich-dark-bg immich-scrollbar"
+	class="grid md:grid-cols-[250px_auto] grid-cols-[70px_auto] relative pt-[4.25rem] h-screen bg-immich-bg dark:bg-immich-dark-bg immich-scrollbar"
```

Comment:
Using pixels for the grid layout means that zooming in and out will lead to issues, since Tailwind heavily uses relative units (rem). But if it's too difficult to change, no problem.

Reply:
I don't think thats something i can change easy, as i oriented myself on the 250px original sidebar size and just resized it to 70px. If you have an idea how to implement that in rem, i would be pleased to try my best on it. For now, i think its too difficult.